types/adobe__pdfservices-node-sdk/operation/delete-pages-operation.d.ts

```diff
@@ -0,0 +1,67 @@
+import { ExecutionContext, FileRef, PageRanges } from '../pdfservices-sdk';
+
+/**
+ * An operation to delete pages in a PDF file.
+ *
+ * Sample Usage:
+ * <pre class="prettyprint">
+ * <code>
+ *  const credentials = PDFServicesSdk.Credentials.serviceAccountCredentialsBuilder()
+ *            .fromFile("pdfservices-api-credentials.json")
+ *            .build(),
+ *        executionContext = PDFServicesSdk.ExecutionContext.create(credentials),
+ *        DeletePages = PDFServicesSdk.DeletePages,
+ *        deletePagesOperation = DeletePages.Operation.createNew(),
+ *        pageRanges = new PDFServicesSdk.PageRanges();
+ *
+ *  deletePagesOperation.setInput( PDFServicesSdk.FileRef.createFromLocalFile('~/Documents/deletePagesOperationInput.pdf',
+ *  	DeletePages.SupportedSourceFormat.pdf));
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/adobe__pdfservices-node-sdk/operation/delete-pages-operation.d.ts;
  Line: 18;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.